README.md

```diff
@@ -87,7 +87,7 @@ java_war(
     data_path = "webapp",
     deps = [
         "//external:appengine/java/api",
-        "//external:javax/servlet/api",
+        "@io_bazel_rules_appengine//appengine:javax.servlet.api",
```

Comment:
You made it a neverlink target (https://github.com/bazelbuild/rules_appengine/commit/ac058970bc8d781d04e570358a1fcc073908f0d3), which seemed worth preserving.